cirq-google/cirq_google/devices/coupler_test.py

```diff
@@ -20,8 +20,9 @@ def test_grid_coupler():
     q1 = cirq.GridQubit(1, 2)
     q2 = cirq.GridQubit(2, 2)
     grid_coupler = cg.Coupler(q1, q2)
-    assert grid_coupler.qubit1 == q1
-    assert grid_coupler.qubit2 == q2
+    assert grid_coupler.qubit0 == q1
+    assert grid_coupler.qubit1 == q2
+    assert grid_coupler.qubits == q1, q2
```

Comment:
Consider also checking the implicit ordering, e.g.
```python
assert cg.Coupler(q2, q1).qubits == q1, q2
```

Reply:
Added.